Content.Server/StationGoal/StationGoalSystem.cs

```diff
@@ -0,0 +1,46 @@
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+using Content.Shared.GameTicking;
+using System.Linq;
+
+namespace Content.Server.StationGoal.Systems
+{
+    /// <summary>
+    ///     Station goal is set at round start.
+    ///     Admin can change the station goal via <see cref="StationGoalCommand"></see>
+    /// </summary>
+    public class StationGoalSystem : EntitySystem
+    {
+        [Dependency] private readonly StationGoalPaperSystem _stationGoalPaperSystem = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+            SubscribeLocalEvent<RoundStartedEvent>(OnRoundStarted);
+        }
+
+        private void OnRoundStarted(RoundStartedEvent ev)
+        {
+            CreateRandomStationGoal();
+        }
+
+        public void CreateRandomStationGoal()
+        {
+            var random = IoCManager.Resolve<IRobustRandom>();
+            var availableGoals = _prototypeManager.EnumeratePrototypes<StationGoalPrototype>();
+            var goalIndex = random.Next(availableGoals.Count() - 1);
+            var goal = availableGoals.ElementAt(goalIndex);
```

Comment:
Можно просто через `Pick`
```suggestion
            var goal = random.Pick(goals.ToList());
```